libs/utils/executor.py

```diff
@@ -252,6 +252,14 @@ def _setup_cpufreq(self, tc):
 
         self.target.cpufreq.set_all_governors(cpufreq['governor'])
 
+        if 'freqs' in cpufreq:
+            if cpufreq['governor'] != 'userspace':
+                raise ValueError('Must use userspace governor to set CPU freqs')
+            logging.info(r'%14s - CPU frequencies: %s',
+                    'CPUFreq', str(cpufreq['freqs']))
+            for cpu, freq in cpufreq['freqs'].iteritems():
```

Comment:
I think it's totally valid to not set frequencies for all CPUs - in fact that's how I've used it in my [use case](https://github.com/bjackman/lisa/blob/fie-test/tests/eas/load_tracking.py#L106)